spec/factories.rb

```diff
@@ -0,0 +1,15 @@
+FactoryBot.define do
+  factory :user do
+    name  { 'User test' }
```

Comment:
not required to change, but if you want you could take a look over [Faker](https://github.com/faker-ruby/faker) for mocking this data